internal/controller/restfuncs.go

```diff
@@ -99,8 +99,12 @@ func commandFunc(w http.ResponseWriter, req *http.Request) {
 	if appErr != nil {
 		http.Error(w, fmt.Sprintf("%s %s", appErr.Message(), req.URL.Path), appErr.Code())
 	} else if event != nil {
-		w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
-		json.NewEncoder(w).Encode(event)
+		if len(event.Readings[0].BinaryValue) > 0 {
+			w.Header().Set(clients.ContentType, clients.ContentTypeCBOR)
```

Comment:
I was CBOR encoding the event here.  I will identify where this should be performed and likely implement via contract client.

Reply:
Got it.  No problem.